README.md

```diff
@@ -63,6 +63,18 @@ When your process starts an AWS SDK library will normally use a chain of credent
 ## Deploying to Kubernetes
 Please see the `deploy` directory for example manifests for deploying to Kubernetes.
 
+There are two options for deployment of `kiam-server`, a `Deployment` and a `DaemonSet`. Some users have found the termination handling of the `DaemonSet` pods causes issues when cycling master nodes (see https://github.com/uswitch/kiam/issues/245). Only deploy **one** of the two `server.*.yaml` files, not _both_.
+
+```
+kubectl apply -f deploy/server.deployment.yaml
+kubectl apply -f deploy/service.yaml
+```
+or
+```
+kubectl apply -f deploy/server.daemonset.yaml
+kubectl apply -f deploy/service.yaml
+```
```

Comment:
I would probably avoid directly telling people to kubectl apply these manifests, they were intended as examples. They're not for production use, like they use latest as the tag and potentially need you to change some things such as where you mount certs in from. So I think I'd just call out that there are two ways to deploy and you can find them in the deploy folder if you want examples

Reply:
Not that I disagree, but this does contextually fit with what was already here.

Reply:
Removed